src/Directory.Build.props

```diff
@@ -23,6 +23,14 @@
     <Nullable>enable</Nullable>
     <ImplicitUsings>enable</ImplicitUsings>
     <PackageReadmeFile Condition="'$(IsPackable)' == 'true'">README.md</PackageReadmeFile>
+
+    <!--
+      Strong name signing of netstandard assemblies for better cross-runtime compatibility
+
+        Ref: https://docs.microsoft.com/en-us/dotnet/core/compatibility/core-libraries/6.0/strong-name-signing-exceptions#recommended-action
+     -->
+    <SignAssembly>true</SignAssembly>
+    <CustomBeforeMicrosoftCommonTargets>$(MSBuildThisFileDirectory)_SignAssembly.targets</CustomBeforeMicrosoftCommonTargets>
```

Comment:
MSBuild project system with all these magic hidden properties and targets overriding each other is really hard to learn 😕

Reply:
agreed